### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ViktorTsupa**
- [DEMO LINK](https://ViktorTsupa.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://ViktorTsupa.github.io/layout_search-bar-airbnb/report/html_report/)

- [x] Icon implemented using background-image CSS property
- [x] Inputs are written inside of 'form' tag with correctly passed attributes
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

it seems that the figma layout or the text in it has errors, because visually everything passes except for one test

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,76 @@
...
+  font-weight: 300;
+  line-height: normal;
+  font-style: normal;
+  color: #3d4e61;
+  margin: 20px 8px;
+  background: #fff;
+}
+
+.input {
+  font-family: Avenir-book, sans-serif;
```
Comment (OleziO) [inline src/style.css:30]:
```suggestion
```

**Review by OleziO — CHANGES_REQUESTED** [review]
image
Fix test

File: src/style.css (hunk 1 above)
Comment (ViktorTsupa) [reply src/style.css:30]: changed, the error is the same
Screenshot_1
Screenshot_2
Screenshot_3

Comment (ViktorTsupa) [reply src/style.css]: Screenshot_1

**Review by lerastarynets — APPROVED** [review]
lgtm 🚀

**Pull request closed without merging**